hibernate-reactive-core/src/main/java/org/hibernate/reactive/id/impl/TableReactiveIdentifierGenerator.java

```diff
@@ -227,17 +227,17 @@ protected int determineIncrement(Properties params) {
 
 	protected String buildSelectQuery() {
 		return "select tbl." + valueColumnName + " from " + renderedTableName + " tbl"
-				+ " where tbl." + segmentColumnName + "=?";
+				+ " where tbl." + segmentColumnName + "=$1";
 	}
 
 	protected String buildUpdateQuery() {
-		return "update " + renderedTableName + " set " + valueColumnName + "=?"
-				+ " where " + valueColumnName + "=?  and " + segmentColumnName + "=?";
+		return "update " + renderedTableName + " set " + valueColumnName + "=$1"
+				+ " where " + valueColumnName + "=$2  and " + segmentColumnName + "=$3";
 	}
 
 	protected String buildInsertQuery() {
 		return "insert into " + renderedTableName + " (" + segmentColumnName + ", " + valueColumnName + ") "
-				+ " values (?, ?)";
+				+ " values ($1, $2)";
 	}
```

Comment:
These changes don't look right: `TableReactiveIdentifierGenerator` isn't specific to Postgres.

Reply:
Yes, you are right. I forgot to check that we are actually on Postgres before parsing. I'm working on it